src/layouts/ReviewRequest/Dashboard.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+  HStack,
+  VStack,
+  Text,
+  Box,
+  Avatar,
+  Flex,
+  Spacer,
+  useClipboard,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  PopoverBody,
+  PopoverArrow,
+} from "@chakra-ui/react"
+import { IconButton } from "@opengovsg/design-system-react"
+import {
+  MenuDropdownButton,
+  MenuDropdownItem,
+} from "components/MenuDropdownButton"
+import { useState } from "react"
+import { BiLink } from "react-icons/bi"
+
+import { extractInitials, getDateTimeFromUnixTime } from "utils"
+
+import { RequestOverview, EditedItemProps } from "./components/RequestOverview"
+
+export interface ReviewRequestDashboardProps {
+  siteName: string
+  reviewUrl: string
+  title: string
+  requestor: string
+  reviewers: string[]
+  changedItems: EditedItemProps[]
+}
+export const ReviewRequestDashboard = ({
+  siteName,
+  reviewUrl,
+  title,
+  requestor,
+  reviewers,
+  changedItems,
+}: ReviewRequestDashboardProps): JSX.Element => {
+  const { onCopy, hasCopied } = useClipboard(reviewUrl)
+
+  return (
+    <Box bg="white" w="100%" h="100%">
+      <VStack
+        bg="blue.50"
+        pl="9.25rem"
+        pr="2rem"
+        pt="2rem"
+        pb="1.5rem"
+        spacing="0.75rem"
+        align="flex-start"
+      >
+        <Flex w="100%">
+          <HStack spacing="0.25rem">
+            <Text textStyle="h5">{title}</Text>
+            {/* Closes after 1.5s and does not refocus on the button to avoid the outline */}
+            <Popover returnFocusOnClose={false} isOpen={hasCopied}>
+              <PopoverTrigger>
+                <IconButton
+                  icon={<BiLink />}
+                  variant="clear"
+                  aria-label="link to pull request"
+                  onClick={onCopy}
+                />
+              </PopoverTrigger>
+              <PopoverContent
+                bg="background.action.alt"
+                _focus={{
+                  boxShadow: "none",
+                }}
+                w="fit-content"
+              >
+                <PopoverArrow bg="background.action.alt" />
+                <PopoverBody>
+                  <Text textStyle="body-2" color="text.inverse">
+                    Link copied!
+                  </Text>
+                </PopoverBody>
+              </PopoverContent>
+            </Popover>
+          </HStack>
+          <Spacer />
+          <ApprovalButton />
+        </Flex>
+        <SecondaryDetails
+          requestor={requestor}
+          reviewers={reviewers}
+          reviewRequestedTime={new Date()}
```

Comment:
Should we get the requested time from the props instead?

Reply:
good catch!